packages/orbit-components/src/Drawer/Drawer.stories.tsx

```diff
@@ -509,31 +396,32 @@ export const SmartFaqSearchInRtl = ({
         </Stack>
       </Drawer>
     </RenderInRtl>
-  );
-};
-
-SmartFaqSearchInRtl.story = {
-  name: "SmartFAQ Search in RTL",
+  ),
 
-  parameters: {
-    info: "Visit Orbit.Kiwi for more detailed guidelines.",
+  args: {
+    title: "Help",
+    fixedHeader: false,
   },
 };
 
-SmartFaqSearchInRtl.args = {
-  shown: true,
-  dataTest: "test",
-  position: POSITIONS.RIGHT,
-  width: "480px",
-  suppressed: true,
-  fixedHeader: false,
-};
+export const Playground: Story = {
```

Comment:
Do we need this story? How is it different from e.g. "With Title"?

Reply:
I just wanted to keep `Playground` story for most of the components. But it does not make sense to have it for this specific component. 💯